packages/fhir-converter/src/templates/cda/References/DiagnosticReport/assessment.hbs

```diff
@@ -25,10 +25,12 @@
   // -------------------------------------------------------------------------------------------------
 --}}
 {
+    "fullUrl":"urn:uuid:{{ID}}",
     "resource":{
         "resourceType": "DiagnosticReport",
         "id":"{{ID}}",
-        "code":{{>DataType/CodeableConcept.hbs code=diagReport.code text=diagReport.title._}},
+        "code":{{>DataType/CodeableConcept.hbs code=diagReport.code text=diagReport.title._ canBeUnknown=true}},
+        "status":{{>ValueSet/DiagnosticReportStatus.hbs code=diagReport.statusCode.code}},
```

Comment:
`DiagnosticReport.status: minimum required = 1, but only found 0` - also added this to every instance of diagnostic report. Code had this issue as well